src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -20,6 +20,12 @@ namespace System.Security.Cryptography.Xml
 {
     internal class Utils
     {
+        // The maximum number of characters in an XML document (0 means no limit).
+        internal const int MaxCharactersInDocument = 0;
+
+        // The entity expansion limit. This is used to prevent entity expansion denial of service attacks.
+        internal const long MaxCharactersFromEntities = (long)1e7;
```

Comment:
0x1e7 is 487 (decimal).
1e7 is 10000000 (decimal).

Please don't put an 0x in front of it :smile:.